Content.Server/Xenoarchaeology/Equipment/Systems/ArtifactAnalyzerSystem.cs

```diff
@@ -491,16 +497,26 @@ private void OnItemRemoved(EntityUid uid, ArtifactAnalyzerComponent component, r
 
     private void OnAnalyzeStart(EntityUid uid, ActiveArtifactAnalyzerComponent component, ComponentStartup args)
     {
-        if (TryComp<ApcPowerReceiverComponent>(uid, out var powa))
-            powa.NeedsPower = true;
+        if (!TryComp<ApcPowerReceiverComponent>(uid, out var powa))
```

Comment:
Frontier comment kind of missing

Reply:
Thanks, still learning the ropes, seems like your commits address the comments so marking as resolved.